cli/formatters/pot.js

```diff
@@ -99,18 +99,49 @@ module.exports = function( matches, options ) {
 
 	output += '\n';
 
-	output += matches.map( function( match ) {
-		var matchPotStr = "",
-			matchId = uniqueMatchId( match );
+	matches.map( function( match ) {
```

Comment:
We do an extra round here because we need to aggregate the lines comments before we start outputting.